### PR TITLE
14408: Drop PROCHLO.

### PR DESCRIPTION
**Pull request by iefremov**
Fixes https://github.com/brave/brave-browser/issues/14408

Resolves 

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Review by fmarier — APPROVED** [review]
`script/brave_license_helper.py` changes look good.

**Comment by iefremov** [comment]
Windows build failed to upload files to S3, tests passes normally

File: components/p3a/p3a_message.cc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  base::Time::Exploded exploded;
+  meta.date_of_survey.LocalExplode(&exploded);
+  DCHECK_GE(exploded.year, 999);
+  const std::string yos = base::NumberToString(exploded.year).substr(2, 4);
+  meta.date_of_install.LocalExplode(&exploded);
+  DCHECK_GE(exploded.year, 999);
+  const std::string yoi = base::NumberToString(exploded.year).substr(2, 4);
+
+  const std::string metastring =
+      "," + meta.country_code + "," + meta.platform + "," + meta.version + "," +
```
Comment (goodov) [inline components/p3a/p3a_message.cc:52]: `base::StrCat` would be nice

File: components/p3a/p3a_message.cc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      meta.channel + "," + yoi + base::NumberToString(meta.woi) + "," + yos +
+      base::NumberToString(meta.wos) + "," + meta.refcode + ",";
+
+  const std::string metric_value_str = base::NumberToString(metric_value);
+
+  // TODO(iefremov): replace with 'if'?
```
Comment (goodov) [inline components/p3a/p3a_message.cc:58]: `CHECK` is fine I guess?

**Review by goodov — APPROVED** [review]

File: components/p3a/p3a_message.cc (hunk 2 above)
Comment (iefremov) [reply components/p3a/p3a_message.cc:58]: so far so good

File: components/p3a/p3a_message.cc (hunk 1 above)
Comment (iefremov) [reply components/p3a/p3a_message.cc:52]: I'll drop the entire function soon, we will upload normal JSONs

**Pull request merged**